org.lflang.lds/uf.py

```diff
@@ -0,0 +1,227 @@
+"""This script updates individual class files in a language
+and diagram server fat jar that has already been created.
+"""
+
+import os
+import argparse
+import subprocess
+import time
+import threading
+import tempfile
+import shutil
+
+# This is the path to the fat jar that is to be updated.
+FAT_JAR = os.path.join('vscode-extension', 'ls', 'lflang-lds.jar')

```

Comment:
This directory no longer exists. This also confuses me, because the `gradle.build` that existed in the `vscode-extension` directory was copying jars from a different location in the tree into `vscode-extension/lf/lflang-lds.jar`. This suggests that `uf.py` is executed _after_ the gradle build script moves stuff into this location. But where is it called from?

Reply:
Tagging @a-sr and @petervdonovan for visibility.

Reply:
@petervdonovan explained that it was a utility script that has outlived its usefulness. I've removed it.